recipes/cosolvkit/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "cosolvkit" %}
+{% set version = "0.4.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/cosolvkit-{{ version }}.tar.gz
+  sha256: 18faa2288d9b2b29f7701c1bde6816c87e2d785a773ba31c8f8932747509d023
+
+build:
+  noarch: python
```

Comment:
This cannot be noarch unfortunately because you are using the `scripts` argument in your setup.py.

Ref: https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python
https://github.com/forlilab/cosolvkit/blob/430e5f40fb0e50262efbfb8a81ef4b1814b0ad6a/setup.py#L27

Reply:
Hi, thanks for the review! How should I modify the recipe to make it work?

thanks!

Reply:
You will have to remove the Python versions and remove the noarch line. Then it will be built as a normal arch package.
The entry points themselves are not sufficient because you still have a scripts tag. If you have only entry points it works but the entry points + scripts tag makes it impossible.

Reply:
I just did remove the no arch line and the Python versions but now the build fails because some packages dependencies need python to be > 3.8 and < 3.12.

Any suggestion on how can I fix this recipe?

Thank you very much for your help!

Reply:
Addressed reviewer's comments.